docs/docsource/operation/XDM_GUI/location_management/add_a_transient_source_storage_location.rst

```diff
@@ -0,0 +1,53 @@
+.. _Add a Transient Source Storage Location:
+
+Add a Transient Source Storage Location
+=======================================
+
+A transient source location is a temporary buffer to which data is stored and
+from which data is replicated. Scality RING with sproxyd is the only
+production-ready environment that supports the transient source replication
+feature (the Zenko Local environment also supports this feature, but is suitable
+for testing purposes only). Data written to the transient source location can be
+replicated to any cloud service Zenko supports.
+
+Adding a transient source storage location is quite similar to adding any other
+storage location.
+
+To deploy a transient source storage location:
+
+#. Follow the instructions in :ref:`Add a Storage Location`, selecting an
+   account name, clicking the **Locations** tab, and clicking **Create
+   Location** to raise the **ADD NEW STORAGE LOCATION** window.
+
+   .. image:: ../../Graphics/xdm_ui_add_new_storage_location.png
+
+#. The **Add New Storage Location** modal appears. Enter the **Location Name**
+   (AWS S3 naming rules apply: lowercase alphanumerics and hyphens only) and
+   select **Scality RING with Sproxyd Connector** from the **Location Type**
+   drop-down list,
+
+   .. image:: ../../Graphics/xdm_ui_add_storage_location_sproxyd.png
```

Comment:
apparently this image breaks the build

Reply:
yeah...but WHY?

Reply:
oh... heh... I hadn't committed and pushed the new screengrabs.